### DISCUSSION TITLE
Axis definitions - Which directions are the Y and Z axes with respect to electrode geometry?

### DISCUSSION THREAD

**Discussion by abhishek-appana** [pybamm-team/PyBaMM#4387] [Q&A]
Hello everyone,

The following question is in the context of pouch cell geometry. I understand the X-dimension corresponds to the thickness of the cell.

Taking reference from the NCA_Kim2011 parameters file, does the electrode height refer to the Z-axis dimension, and does the electrode width refer to the Y-axis dimension? Could someone please confirm this for me?

Thank you

**Comment by valentinsulzer** [comment]
This is correct. Although it only really matters if you are setting the option "dimensionality" to 2.
